blazor/diagram/scroll-settings.md

```diff
@@ -101,7 +101,92 @@ The Diagram control provides the following event for the scroll settings.
 
 }
 ```
+## AutoScroll 
+The auto-scroll feature automatically scrolls the diagram whenever the node is moved beyond its boundaries. Auto-scroll is enabled during node dragging and  node resizing, and multiple-selection operations.
```

Comment:
The auto-scroll feature automatically scrolls the diagram whenever the node is moved beyond its boundaries. Auto-scroll is enabled during node dragging, node resizing, and multiple-selection operations.